components/org.wso2.bfsi.consent.management.service/src/main/java/org/wso2/bfsi/consent/management/service/util/TokenRevocationUtil.java

```diff
@@ -37,8 +41,8 @@ public static void revokeTokens(DetailedConsentResource detailedConsentResource,
         AuthenticatedUser authenticatedUser = getAuthenticatedUser(userID);
         Set<AccessTokenDO> accessTokenDOSet = getAccessTokenDOSet(detailedConsentResource, authenticatedUser);
 
-        //TODO : Get consent id claim name from configuration
-        String consentIdClaim = "consent_id";
+        String consentIdClaim = ConsentManagementConfigParser.getInstance().getConfiguration()
+                .get(ConsentManagementConstants.CONSENT_ID_CLAIM_NAME).toString();
```

Comment:
Isn't there a possibility to NPE?

Reply:
Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/12/commits/0efffb1b9f4a14998bc2841b8ec470d8aa5ce755